lib/impress.js

```diff
@@ -191,54 +190,50 @@ impress.detectAppDir = (
   let dir = impress.applicationsDir + '/' + appName;
   const linkFile = dir + '/application.link';
```

Comment:
Why do we support these `.link` files if the same behavior can be achieved using regular symlinks? Windows has symlinks too, since introduction of NTFS 😉